embc-app/DataInterfaces/ModelBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using Gov.Jag.Embc.Public.Models.Db;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Gov.Jag.Embc.Public.DataInterfaces
+{
+    public static class ModelBuilderExtensions
```

Comment:
Can we merge all extension methods to a single Ex file or merge into the context file instead? Is using extension methods a best practice for this?

Reply:
We can definitely merge the the logic from the SaveChanges directly into the override.  As for the ModelBuilderExtensions, I was unsuccessful to get this functionality to work successfully outside of this pattern.